src/prefabs/icon.js

```diff
@@ -0,0 +1,1287 @@
+(() => ({
+  name: 'Icon',
+  icon: 'ContainerIcon',
+  category: 'LAYOUT',
+  structure: [
+    {
+      name: 'Icon',
+      options: [
+        {
+          label: 'Icon',
+          key: 'icon',
+          value: 'None',
+          type: 'CUSTOM',
+          configuration: {
+            as: 'DROPDOWN',
+            dataType: 'string',
+            allowedInput: [
+              {
```

Comment:
I have my doubts of this value. Do our users need a value `None` when they dragged an icon on the canvas? Why did they drag in the first place? And what do they get?